.circleci/config.yml

```
@@ -218,41 +268,33 @@ jobs:
       - run:
           name: Install cargo-llvm-cov
           command: cargo install cargo-llvm-cov
-      # Note: This build can potentially exceed the amount of memory availble to the CircleCI instance.
+      # Note: This build can potentially exceed the amount of memory available to the CircleCI instance.
       # We've seen that limiting the number of jobs helps reduce the frequency of this. (Note that
       # when doing discovery, we found that the docker image `meminfo` and `cpuinfo` often report
       # the machine level memory and CPU which are far higher than the memory allocated to the docker
       # instance. This may be causing rust to be overly greedy triggering the VM to OOM the process.)
       - run:
-          name: Report tests and coverage
+          name: Unit tests
           command: |
-            cargo llvm-cov --summary-only --json --output-path workspace/test-results/cov.json nextest --features=emulator --features=bigtable --jobs=2 --profile=ci
-      - store_artifacts:
-          path: ~/project/workspace/test-results/cov.json
-          destination: cov.json
-      - store_artifacts:
-          path: ~/project/target/nextest/ci/junit.xml
-          destination: junit.xml
+            make unit-test
       - store_test_results:
-          path: target/nextest/ci
+          path: workspace/test-results
       - save_test_cache:
-          cache_key: rust-v2-cache-{{ checksum "Cargo.lock" }}
+          cache_key: rust-v2-cache-{{ checksum "Cargo.lock" }}'
```

```suggestion
          cache_key: rust-v2-cache-{{ checksum "Cargo.lock" }}
```
Not sure if that's meant to be there.